scripts/lc/install-from-dev-pkg.sh

```diff
@@ -2,7 +2,6 @@ trap 'echo "# $BASH_COMMAND"' DEBUG
 
 SPACK_PKG_NAME=${SPACK_PKG_NAME:-'spheral'}
 SPACK_URL=${SPACK_URL:-'https://github.com/spack/spack'}
-BUILD_ALLOC=${BUILD_ALLOC}
```

Comment:
Change this to `BUILD_ALLOC=${BUILD_ALLOC:-''}` To default to an empty call when not defined

Reply:
Actually we might not even need this